src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033033 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033033 "2023-04-04T10:54:00Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033033 "FMB cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033033 "A flat midget bipolar cell expresses the calcium binding protein recoverin.")
+AnnotationAssertion(rdfs:label obo:CL_4033033 "flat midget bipolar cell")
+SubClassOf(obo:CL_4033033 obo:CL_0000750)
+
+# Class: obo:CL_4033034 (invaginating midget bipolar cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(oboInOwl:hasDbXref "PMID:32032773") obo:IAO_0000115 obo:CL_4033034 "An ON bipolar cell with a small dendritic tree that forms most of the central (invaginating) elements opposite the synaptic ribbon at the cone triad.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033034 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033034 "2023-04-04T09:51:44Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033034 "IMB cell")
+AnnotationAssertion(rdfs:label obo:CL_4033034 "invaginating midget bipolar cell")
+SubClassOf(obo:CL_4033034 obo:CL_0000749)
+
+# Class: obo:CL_4033035 (giant bipolar cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033035 "An ON bipolar cell that has large dendritic and axonal fields.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033035 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033035 "2023-04-04T09:54:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033035 "GB cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033035 "A giant bipolar cell can be bistratified or monostratified.")
+AnnotationAssertion(rdfs:label obo:CL_4033035 "giant bipolar cell")
+SubClassOf(obo:CL_4033035 obo:CL_0000749)
+
+# Class: obo:CL_4033036 (OFFx cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31995762") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033036 "An OFF bipolar cell that might be NXPH1-positive and FEZF1-positive.")
```

Comment:
The differentia in the text definition seems insufficient ("might be... "). PMID:30712875 may be helpful in identifying some differential features (e.g., specific to fovea).

Reply:
I agree that a cell type only described by cell expression is insufficient. However, these cells being specific to the fovea is a hypothesis due the lack of expression of the genes in the periphery, but does not respond to any histological observation